core/testcasecontroller/simulation_system_admin/simulation_system_admin.py

```diff
@@ -0,0 +1,96 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+
+def check_host_docker():
+    # check whether Docker is installed on the host.
+    # If Docker is not installed, try to install Docker with one-click installation script.
+    shell_cmd = "docker version | head -n 2"
+    check_docker = subprocess.run(shell_cmd, shell=True)
+
+    if check_docker.returncode != 0:
+        # trying to install docker
+        print("trying to install docker")
+        try:
+            shell_install_docker_cmd = "curl -fsSL https://get.docker.com | bash -s docker --mirror Aliyun"
+            install_docker = subprocess.run(shell_install_docker_cmd, shell=True)
+
+            if install_docker.returncode == 0:
+                print("successfully installed docker")
+            else:
+                raise Exception(f"install docker failed")
+        except Exception as err:
+            raise Exception(f"install docker failed, error: {err}.") from err
+
+    print("check docker successful")
+
+def check_host_kind():
+    # check whether Docker is installed on the host.
+    shell_cmd = "kind version"
+    check_kind = subprocess.run(shell_cmd, shell=True)
+
+    if check_kind.returncode != 0:
+        raise Exception(f"Kind is not installed. You need to install Kind")
+    else:
+        print("check docker successful")
+
+def get_host_free_memory_size():
+    # return the current memory(free) on the host(in kB)
+    shell_cmd = "cat /proc/meminfo | grep MemFree"   # in kB
+    get_memory_info = subprocess.Popen(shell_cmd, shell=True, stdout=subprocess.PIPE)
+    memory_info = get_memory_info.stdout.read()
+    memory_free = int(str(memory_info).split(':')[1].strip().split(' ')[0])
+
+    return memory_free
+
+
+def check_host_memory():
+    # check whether the current memory is sufficient(>=4GB)
+    memory_free = get_host_free_memory_size()
+    memory_require = 4 * 1024 * 1024    # 4GB
+
+    if memory_free < memory_require:
+        print("The current free memory is insufficient. Current Memory Free: {} kB, Memory Require: {} kB".format(memory_free, memory_require))
+        raise Exception(f"The current free memory is insufficient.")
+    else:
+        print('Memory check successful')
+
+
+def check_host_enviroment():
+    check_host_docker()
+    check_host_kind()
+    check_host_memory()
+
+def build_simulation_enviroment(simulation):
+    check_host_enviroment()
+
+    shell_cmd = 'cat /home/iszhyang/program/ianvs/yaml/all_in_one.sh | ' \
```

Comment:
Is this just a test?

Reply:
It is recommended not to use your own local directory path, and create a new issue to deal with the offline installation of sedna and ianvs.

Reply:
Thanks, I have fix it.